app/src/main/java/com/gh4a/fragment/NotificationListFragment.java

```diff
@@ -192,11 +186,40 @@ public void onClick(DialogInterface dialog, int which) {
                         .setNegativeButton(R.string.cancel, null)
                         .show();
                 return true;
+            case R.id.notification_filter_unread:
+                mAll = false;
+                mParticipating = false;
+                item.setChecked(true);
+                reloadNotification();
+                return true;
+            case R.id.notification_filter_all:
+                mAll = true;
+                mParticipating = false;
+                item.setChecked(true);
+                reloadNotification();
+                return true;
+            case R.id.notification_filter_participating:
+                mAll = false;
+                mParticipating = true;
+                item.setChecked(true);
+                reloadNotification();
+                return true;
```

Comment:
This can probably be shortened to:

```
case R.id.notification_filter_unread:
case R.id.notification_filter_all:
case R.id.notification_filter_participating:
    mAll = itemId == R.id.notification_filter_all;
    mParticipating = itemId == R.id.notification_filter_participating;
    item.setChecked(true);
    reloadNotifications();
    return true;
```